sunkit_image/flct/flct.py

```diff
@@ -0,0 +1,499 @@
+        major or column major.
+
+    Returns
+    -------
+    `tuple`
+        A tuple containing two `~numpy.ndarray` arrays.
+    """
+
+    # Checking whether the C extension is correctly built.
+    if _pyflct is None:
+        raise ImportError("C extension for flct is missing, please rebuild.")
+
+    if order.lower() not in ["row", "column"]:
+        raise ValueError("The order of the arrays is not correctly specified. It can only be 'row' or 'column'")
+
+    if order is "row":
+        transp = 0
+    else:
+        transp = 1
+
+    ier, a, b = _pyflct.read_two_images(filename, transp)
+
+    if ier is not 1:
+        raise ValueError("The file was not read correctly. Please check the file")
+
+    else:
+        return a, b
+
+
+def read_3_images(filename, order="row"):
+    """
+    A python function to read three arrays of the same size from ``dat`` files.
+
+    .. note ::
+        This function can be used to read only special arrays which were written
+        using the ``write`` functions in `~sunkit_image.flct` or the IDL IO routines
+        as given on the FLCT source `website <http://solarmuri.ssl.berkeley.edu/~fisher/public/software/FLCT/C_VERSIONS/>`__.
+
+    Parameters
+    ----------
+    filename : `string`
+        The name of ``dat`` file from where the arrays are to be read.
+    order : {"row" | "column"}
+        The order in which the array elements are stored that is whether they are stored as row
+        major or column major.
+
+    Returns
+    -------
+    `tuple`
+        A tuple containing three `~numpy.ndarray` arrays.
+    """
+
+    # Checking whether the C extension is correctly built.
+    if _pyflct is None:
+        raise ImportError("C extension for flct is missing, please rebuild.")
+
+    if order.lower() not in ["row", "column"]:
+        raise ValueError("The order of the arrays is not correctly specified. It can only be 'row' or 'column'")
+
+    if order is "row":
+        transp = 0
+    else:
+        transp = 1
+
+    ier, a, b, c = _pyflct.read_three_images(filename, transp)
+
+    if ier is not 1:
+        raise ValueError("The file was not read correctly. Please check the file")
+
+    else:
+        return a, b, c
+
+
+def write_2_images(filename, array1, array2, order="row"):
+    """
+    A python function to write two arrays of the same size to a ``dat`` file.
+
+    Parameters
+    ----------
+    filename : `string`
+        The name of ``dat`` file in which the arrays are to be stored.
+    array1 : `numpy.ndarray`
+        The first array to be stored in the dat file.
+    array2 : `numpy.ndarray`
+        The second array to be stored in the dat file.
+    order : {"row" | "column"}
+        The order in which the array elements are stored that is whether they are stored as row
+        major or column major.
+
+    Returns
+    -------
+    A dat file of the ``filename`` is created in the current directory without returning anything.
```

Comment:
The function doesn't technically return anything.

Reply:
yes that is true. should I remove the return part?

Reply:
Yes please.